extensions/wikia/WallNotifications/WallNotifications.class.php

```diff
@@ -537,7 +533,6 @@ protected function addNotificationLink( $userId, WallNotificationEntity $notific
 				'entity_key' => $notification->id,
 				'author_id' => $notifData->msg_author_id,
 				'is_reply' => $notification->isReply(),
-				'notifyeveryone' => $notifData->notifyeveryone
```

Comment:
I've added a default value to the corresponding DB column, so we can now omit this in INSERTs (and drop the column post release).